modules/home-manager/ghostty.nix

```
@@ -0,0 +1,24 @@
+{ catppuccinLib }:
+{ config, lib, ... }:
+
+let
+  inherit (config.catppuccin) sources;
+
+  cfg = config.catppuccin.ghostty;
```

```suggestion
  cfg = config.catppuccin.ghostty;
  enable = cfg.enable && programs.ghostty.enable;
```
Since we're also placing a file in `~/.config` with `xdg.configFile`, we should make sure that is only done when Ghostty itself is enabled (just like the values of `programs.ghostty.settings` only being written to its config file when its enabled)